sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -891,6 +891,74 @@ def test_cat(self):
     self._run_test(lambda s: s.cat.as_unordered(), s)
     self._run_test(lambda s: s.cat.codes, s)
 
+  @parameterized.expand(frames.ELEMENTWISE_DATETIME_PROPERTIES)
+  def test_dt_property(self, prop_name):
+    # Generate a series with a lot of unique timestamps
+    s = pd.Series(
+        pd.date_range('1/1/2000', periods=100, freq='m') +
```

Comment:
Should we add a timezone here too, or does it not matter?

Reply:
I don't think it matters. Some of the properties are ambiguous in this case (example `.day` could be different depending on the timezone), but pandas has made a decision about how to handle these ambiguities, and they allow it. Here we're just verifying we do the same thing as pandas.